### ISSUE TITLE
Incorporating changes in our RakNet fork.

### ISSUE THREAD

**Issue by Luke1410** [larku/RakNet#45]
Hi larku,

we are actively working on further developing RakNet as part of our own fork (https://github.com/SLikeSoft/SLikeNet). This all started around 2-3 years ago and eventually evolved in founding a company under which we continue the development of SLikeNet.
Only after the company was found, I became aware of your fork and realized you made already the attempt to revive RakNet. If that's still of interest to you, I'd be glad to get in touch with you (please contact me via info @ slikesoft . com ).

In addition, I'd like to ask you kindly whether it'd be acceptable to you, if we process the commits you merged already into your fork and review them for eventual integration into our fork. Also note that we already merged one of the commits ( https://github.com/larku/RakNet/commit/c15dd90afb7268f531312bc4b149206f219452b0 ) which a user of our fork pointed us to, since he ran into the issue this commit fixed.

**Comment by Ident** [comment]
Has there been any communication between larku and slikenet since?

**Comment by Luke1410** [comment]
Nope, we didn't hear any word from @larku yet. We're therefore planning to process the commits which were done in this fork and see if anything should still be integrated into SLikeNet.
